### DISCUSSION TITLE
Custom intro screens

### DISCUSSION THREAD

**Discussion by murryguy** [ppy/osu#28145] [Ideas] [answered]
I used to make as a pull request!
 Anyways "heres" some files about it.
![osu_2024-04-28_20-21-44](https://github.com/ppy/osu/assets/167148240/2b138c90-f8ee-4f5d-bb6c-9c1bad02c477)
![osu_2024-04-28_20-22-06](https://github.com/ppy/osu/assets/167148240/5e8762ee-8d97-4181-bd05-7a24d1a427d6)
not my best work

**Comment by Gabixel** [comment]
I don't quite understand what the request is here.
My first guess is that you're asking for custom intros (when you start the app), but since you're trying to show this on the beatmap editor, maybe you're asking for custom beatmap intros? (or an intro for the editor?)

I hope you can give more information

**Reply by murryguy** [reply]
I photoshopped the beatmap editor

Custom app intros

**Comment by Gabixel** [comment, marked as answer]
I could have sworn that this was already requested, but instead it seems to me that only something about being able to choose the beatmap after the intro [(1)](https://github.com/ppy/osu/discussions/15169) [(2)](https://github.com/ppy/osu/discussions/26729) (in addition to putting the [seasonal themes](https://github.com/ppy/osu/discussions/24403) back) was suggested.

I don't know if the devs want to allow that or if it's too out of scope, but I could see the possibility of doing it with the skin editor (and, at that point, maybe having the intros "baked" into the skin), but that's probably something for the distant future.

**Comment by murryguy** [comment]
> I could have sworn that this was already requested, but instead it seems to me that only something about being able to choose the beatmap after the intro [(1)](https://github.com/ppy/osu/discussions/15169) [(2)](https://github.com/ppy/osu/discussions/26729) (in addition to putting the [seasonal themes](https://github.com/ppy/osu/discussions/24403) back) was suggested.
> 
> I don't know if the devs want to allow that or if it's too out of scope, but I could see the possibility of doing it with the skin editor (and, at that point, maybe having the intros "baked" into the skin), but that's probably something for the distant future.

O.K.
